acra-censor/handlers/whitelist_handler.go

```diff
@@ -216,17 +216,25 @@ func (handler *WhitelistHandler) Release() {
 	handler.Reset()
 }
 
-// AddQueries adds queries to the list that should be whitelisted
+// AddQueries normalizes and adds queries to the list that should be whitelisted
 func (handler *WhitelistHandler) AddQueries(queries []string) {
 	for _, query := range queries {
-		handler.queries[query] = true
+		normalizedQuery, _, err := NormalizeAndRedactSQLQuery(query)
```

Comment:
same as with blacklist. I think we should use non-redacted query

Reply:
it's not redacted, it's normalized ;)